tests/admin/import/test-class-import-aioseo.php

```diff
@@ -95,6 +159,26 @@ public function test_cleanup() {
 		$this->assertEquals( $this->status( 'cleanup', true ), $result );
 	}
 
+	/**
+	 * @covers WPSEO_Import_AIOSEO::__construct
+	 * @covers WPSEO_Import_AIOSEO::run_cleanup
+	 * @covers WPSEO_Import_AIOSEO::cleanup
+	 */
+	public function test_cleanup_gone_bad() {
+		$mock = $this->getMockBuilder( 'wpdb' )
+					 ->setConstructorArgs( array( DB_USER, DB_PASSWORD, DB_NAME, DB_HOST ) )
+					 ->setMethods( array( 'query' ) )
+					 ->getMock();
+		$mock->expects( $this->any() )
+			 ->method( 'query' )
```

Comment:
See remark above about the indentation - in this case, I currently see `tab - space`, while only `tab` is expected/allowed.